File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/moonlight/MoonlightFragment.java
package com.art2cat.dev.moonlightnote.controller.moonlight;

import com.google.firebase.database.DatabaseReference;
import com.google.firebase.database.Query;

/**
 * Created by rorschach.h on 8/13/16.
 */
public class MoonlightFragment extends MoonlightListFragment {

  @Override
  public Query getQuery(DatabaseReference databaseReference) {
    return databaseReference.child("users-moonlight")
        .child(getUid()).child("note");
  }

  @Override
  public boolean isTrash() {
    return false;
  }
}

File: app/src/main/java/com/art2cat/dev/moonlightnote/controller/moonlight/MoonlightFragment.kt
package com.art2cat.dev.moonlightnote.controller.moonlight

import com.google.firebase.database.DatabaseReference
import com.google.firebase.database.Query

/**
 * Created by Rorschach
 * on 24/05/2017 8:01 PM.
 */
class MoonlightFragment() : MoonlightListFragment() {
    override val isTrash: Boolean
        get() = false

    override fun getQuery(databaseReference: DatabaseReference): Query {
        return databaseReference.child("users-moonlight")
                .child(uid).child("note")
    }
}